[2024-09-25T08:51:04.818+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T08:51:04.945+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.dellstore_db.extract.inventory scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T08:51:05.017+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.dellstore_db.extract.inventory scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T08:51:05.018+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-25T08:51:05.129+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): dellstore_db.extract.inventory> on 2024-09-24 00:00:00+00:00
[2024-09-25T08:51:05.204+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=344) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T08:51:05.218+0000] {standard_task_runner.py:72} INFO - Started process 365 to run task
[2024-09-25T08:51:05.257+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'dellstore_db.extract.inventory', 'scheduled__2024-09-24T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpbfvrow2z']
[2024-09-25T08:51:05.265+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask dellstore_db.extract.inventory
[2024-09-25T08:51:06.073+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.dellstore_db.extract.inventory scheduled__2024-09-24T00:00:00+00:00 [running]> on host 857abb7f2bfc
[2024-09-25T08:51:06.897+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='dellstore_db.extract.inventory' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:00:00+00:00'
[2024-09-25T08:51:06.904+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T08:51:07.045+0000] {base.py:84} INFO - Retrieving connection 'dellstore_db'
[2024-09-25T08:51:07.814+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T08:51:07.903+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T08:51:07.936+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T08:51:07.937+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=dellstore_db.extract.inventory, run_id=scheduled__2024-09-24T00:00:00+00:00, execution_date=20240924T000000, start_date=20240925T085104, end_date=20240925T085107
[2024-09-25T08:51:08.161+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T08:51:08.288+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-25T08:51:08.300+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
